tools/cldr-apps/src/main/webapp/AdminPanel.jsp

```diff
@@ -9,6 +9,11 @@ if(vap==null ||
 	response.sendRedirect("http://cldr.unicode.org"); // Get out.
 	return;
 }
+if (SurveyTool.useDojo(request) == false) {
+    // AdminPanel.jsp is the dojo version. Redirect to the new one.
+    response.sendRedirect("./v?USE_DOJO=false#admin///");
```

Comment:
this jsp is in admin.html and other places, so it's worth having this redirect here